### DISCUSSION TITLE
What is the water-phase pressure P_w in the capillary pressure

### DISCUSSION THREAD

**Discussion by WilkAndy** [idaholab/moose#17519] [Q&A Modules: General] [answered]
(This message was posted on behalf of samuel171228 who posted the original question on the defunct google-groups.)

Hello all MOOSE user

I'm using a porous flow module to make single-phase, unsaturated flow in porous media.
To calculate saturation and relative permeability.
But I have some problem with the definition of the capillary pressure

I think When we calculate capillary pressure(P_c), the P_c is the difference between water-phase pressure and gas-phase pressure
-> P_c = - P_w
(because of single-phase & unsaturated flow)

In here, what is the P_w?
Is it pore-pressure or not?
(In my input file, the pore-pressure can be calculated as hydrostatic pressure)

**Comment by WilkAndy** [comment, marked as answer]
Hi Samuel,

Welcome to MOOSE and PorousFlow.  You are correct that P_c = -P_w, and that P_w is porepressure (in your single-phase case).  Note that if P_w >= 0 then the system is fully saturated (S_w = 1) and so P_c has no physical meaning (in your single-phase case).  When P_w < 0 then the system is unsaturated (S_w < 1) and P_c > 0, and P_c has a physical meaning.

a

**Reply by WilkAndy** [reply]
Without further input from Samuel, i'm marking this as "answered".

**Reply by samuel171748** [reply]
Oh Andy, Thank you for your help!
